tools/protovalidate-conformance/internal/results/set.go

```diff
@@ -53,17 +54,20 @@ func (set *Set) Print(w io.Writer) {
 		(*SuiteResults)(suite).Print(writer)
 	}
 	res := resultLabel(set.Failures == 0)
-	_, _ = fmt.Fprintf(writer, "%s (failed: %d, passed: %d, total: %d)\n",
-		res, set.Failures, set.Successes, set.Failures+set.Successes)
+	_, _ = fmt.Fprintf(writer, "%s (failed: %d, skipped: %d, passed: %d, total: %d)\n",
+		res, set.Failures, set.Skipped, set.Successes, set.Failures+set.Successes+set.Skipped)
 	_ = writer.Flush()
 }
 
 type SuiteResults harness.SuiteResults
 
 func (suite *SuiteResults) AddCase(res *harness.CaseResult, verbose bool) {
-	if res.Success {
+	switch {
+	case res.Success && !res.Skipped:
 		suite.Successes++
-	} else {
+	case !res.Success && res.Skipped:
```

Comment:
Is this not the equivalent of: **if failure && skipped?**

Should it not be just: **if skipped**

Reply:
what about when skipped and successful? that is a failure.

Reply:
Humm, interesting, so "skipped" tests are actually run?

Then `--skipped` is probably the wrong name, the flag should be `--expected-failures` or something like that.